starters/tailwind/stories/Disclosure.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta } from "@storybook/react";
+import React from "react";
+import {
+  Disclosure,
+  DisclosureHeader,
+  DisclosurePanel,
+  DisclosureGroup,
+} from "../src/Disclosure";
+
+const meta: Meta<typeof Disclosure> = {
```

Comment:
Not sure why controls aren't being generated automatically in storybook...

Reply:
hm i see the controls in the storybook